app/controllers/concerns/attachments_copier.rb

```diff
@@ -0,0 +1,19 @@
+module AttachmentsCopier
+  def copy_attachments(record)
+    record.content.scan(Attachment::SCREENSHOT_REGEX).each do |screenshot_url|
+      _, _, _, _, project_id, node_id, filename, _ = screenshot_url
+
+      attachment = Attachment.find_by(filename: filename, node_id: record.node_id_was)
```

Comment:
We'll need to escape the filename like we do in attachment_validator https://github.com/secroots/dradis-pro/blob/2d14058431f2760904edbb2d74bfb83d4c7a12b8/engines/dradispro-word/lib/dradis/pro/plugins/word/validators/attachment_validator.rb#L108